### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'find'

### ISSUE THREAD

**Issue by evandrocoan** [wbond/package_control#1211]
### AttributeError: 'NoneType' object has no attribute 'find'

When I am using this sublime-workspace

[Galileo_DEVELOPMENT.sublime-workspace.txt](https://github.com/wbond/package_control/files/735960/Galileo_DEVELOPMENT.sublime-workspace.txt)

, I and I try to install a package or disable one, it gives this error:
```
Traceback (most recent call last):
  File "D:\User\Dropbox\Applications\SoftwareVersioning\SublimeText\Data\Packages\Package Control\package_control\commands\disable_package_command.py", line 50, in on_done
    self.disable_packages(package, 'disable')
  File "D:\User\Dropbox\Applications\SoftwareVersioning\SublimeText\Data\Packages\Package Control\package_control\package_disabler.py", line 119, in disable_packages
    if syntax.find('Packages/' + package + '/') != -1:
AttributeError: 'NoneType' object has no attribute 'find'
```

I did a complete clean install, and reinstalled all the packages, but even after that, If I put that `Session.sublime_session`, the package install does not work.

If I delete it, and let Sublime Text create a new one, it works fine to install and disable packages.

**Comment by FichteFoll** [comment]
Except for your last opened file, `"file": "/D/User/Dropbox/Applications/SoftwareVersioning/Subtrees/Galileo/data/lang/galileo.txt",`, all files have a syntax override set.

Do you perhaps set your `syntax` setting to `None` in your project settings or in the user settings? Do you have other packages installed that interfere with setting syntaxes for view?

Regardless, Package Control should not panic in this situation.

**Comment by Mockernut** [comment]
For the love of god, stop sending me these emails. And before anyone tells me to go to mute it, I've tried. I've tried to mute, unsubscribe, you name it. I can't get them to STOP!

**Comment by evandrocoan** [comment]
> Do you perhaps set your syntax setting to None in your project settings or in the user settings?

I do not think there is.


> Do you have other packages installed that interfere with setting syntaxes for view?

I have some packages which interfere with syntax set as Color Highlither.

**Comment by wbond** [comment]
This should be fixed by 78a2081
